frontend/src/config/myanmar/layers.json

```diff
@@ -2278,5 +2278,87 @@
     "exposure": {
       "id": "wp_pop_cicunadj"
     }
-  }
+  },
+  "conflict_fatalities": {
+    "title": "Number of fatalities in an armed conflict incident",
+    "loader": "acled",
+    "additional_query_params": {
+      "iso": 104,
+      "limit": 0
+    },
+    "type": "point_data",
+    "data": "http://localhost/acled",
```

Comment:
you need to find a way to have this point to the API somehow. Hence my suggestion to look at #559 ;)